libs/types/models/DeviceRebootHookSpec.ts

```diff
@@ -0,0 +1,14 @@
+/* generated using openapi-typescript-codegen -- do no edit */
+/* istanbul ignore file */
+/* tslint:disable */
+/* eslint-disable */
+import type { HookAction } from './HookAction';
```

Comment:
Type `HookAction` is not being generated correctly and I reverted it manually.

I added a comment in the [PR](https://github.com/flightctl/flightctl/pull/449) which modified it so we can get it fixed soon.